src/elements/block/core-paragraph/core-paragraph.tsx

```diff
@@ -14,7 +14,7 @@ export type CoreParagraphProps = {
   __typename: 'CoreParagraphBlock'
 }
 
-export function CoreParagraph({ attributes }: CoreParagraphProps) {
+export function CoreParagraph({ attributes, __typename }: CoreParagraphProps) {
```

Comment:
this __typename is an unused value.
```suggestion
export function CoreParagraph({ attributes }: CoreParagraphProps) {
```